frontend/packages/dev-console/src/components/catalog/CatalogPage.tsx

```diff
@@ -50,13 +53,18 @@ const CatalogPage: React.FC<CatalogPageProps> = ({ match }) => {
       ) : (
         <CreateProjectListPage title={t('devconsole~Developer Catalog')}>
           {(openProjectModal) => (
-            <Trans t={t} ns="devconsole">
-              Select a Project to view the developer catalog or{' '}
-              <Button isInline variant="link" onClick={openProjectModal}>
-                create a Project
-              </Button>
+            <>
+              {t('devconsole~Select a Project to view the developer catalog')}{' '}
+              {canCreateProject && (
+                <Trans t={t} ns="devconsole">
+                  or{' '}
+                  <Button isInline variant="link" onClick={openProjectModal}>
+                    create a Project
+                  </Button>
+                </Trans>
```

Comment:
I believe you missed using your new component here, or is there a difference that I didn't see?